pages/date-range-picker/month-calendar-permutations.page.tsx

```diff
@@ -52,7 +68,7 @@ export default function DateRangePickerCalendarPage() {
     <Box padding="s">
       <h1>Date-range-picker month calendar page for screenshot tests</h1>
       <ScreenshotArea>
-        <div style={{ blockSize: `${intervals.length * 400}px` }}>
+        <div style={{ blockSize: `${(1 + permutations.length) * 400}px` }}>
```

Comment:
Why not just apply some block-end margin instead of hard-coding the block size?

Reply:
I see, the individual permutation items don't have a block size because they display dropdowns. But applying a block size to each of them instead of a combination of inset-block-size for each item + blockSize for the parent would be a little bit cleaner IMO.